src/components/Composer/index.js

```diff
@@ -193,6 +197,39 @@ class Composer extends React.Component {
         this.textInput.removeEventListener('wheel', this.handleWheel);
     }
 
+    onChangeText(text) {
+        // updates the text input to reflect the current value
+        this.setState({value: text});
+
+        // calls the onChangeText callback prop
+        if (this.props.onChangeText != null) {
+            this.props.onChangeText(text);
+        }
+    }
+
+    onSelectionChange(event) {
```

Comment:
I am leaving these names as it is because it seems they are just wrappers to the original prop callbacks and I don't have better names for these. 

But we prefer
>